docs/GettingStarted.md

```diff
@@ -123,7 +122,19 @@ architecture (powerpc or amd64) from the website and run it.
         ...
 
 
+Note: 
+	1) If there is different OS(other than ONL) running on the swicth.Then halt the booting process at U-boot mode, Then check for the ONIE deatils in the environment(=> printenv). Open the ONIE in rescue mode, while ONIE has many different installation modes, we recommend the rescue mode for doing a manual (read: via console) because it disables the automatic ONIE server discovery. Then run (=> run onie_rescue) command to take you to the ONIE environment.
 
+       2) To load ONL installer from ONIE directly from the devlopement environment,
+       	
+       	run a http server directly from the location (example:python -m SimpleHTTPServer 8000) and access it as,
+	   example: ONIE:/ # install_url http://buildmachine:/path/to/ONL.installer.
+
+Also, you can use install via scp with two steps,
+
+       example: ONIE:/ # scp [username]@buildmachine:/path/to/ONL.installer 
+       		(ONL.installer # update for specific file/date/build)
```

Comment:
Can you put this all on one line?  The `scp` command needs a destination and the "ONL.installer" was supposed to be that destination and remove the parentheses?